cmd/nerdctl/compose_down_linux_test.go

```diff
@@ -60,8 +57,12 @@ services:
 	base.ComposeCmd("-p", projectName, "-f", compFull.YAMLFullPath(), "up", "-d").AssertOK()
 	defer base.ComposeCmd("-p", projectName, "-f", compFull.YAMLFullPath(), "down", "--remove-orphans").AssertOK()
 
-	base.ComposeCmd("-p", projectName, "-f", compOrphan.YAMLFullPath(), "down", "-v").AssertCombinedOutContains("is in use")
-
+	// The error output is different with docker
```

Comment:
Can we fix this?

Reply:
Or just check "in use"

Reply:
> Or just check "in use"

It looks okay. But it may not be clear enough as to the meaning of whether or not.

Reply:
Updated.